stubs/tests/ResourceTest.php

```diff
@@ -57,6 +57,29 @@ public function test_users_can_edit_resources_belonging_to_them()
         $response->assertRedirect(localized_route('resources.show', $resource));
     }
 
+    public function test_users_can_translate_resources_belonging_to_them()
+    {
+        if (! config('hearth.resources.enabled')) {
+            return $this->markTestSkipped('Resource support is not enabled.');
+        }
+
+        $user = User::factory()->create();
+        $resource = Resource::factory()->create(['user_id' => $user->id]);
+
+        $resource->setTranslation('title', 'en', 'title in English');
+        $resource->setTranslation('title', 'fr', 'title in French');
+
+        $this->assertEquals($resource->getTranslation('title', 'en'), 'title in English');
+        $this->assertEquals($resource->getTranslation('title', 'fr'), 'title in French');
+
+        $translationObject = ['en' => 'title in English', 'fr' => 'title in French'];
+
+        $this->assertEquals($resource->getTranslations('title'), $translationObject);
+
+        $this->expectExceptionMessage("Cannot translate attribute `language` as it's not one of the translatable attributes: `title, summary`");
+        $resource->setTranslation('language', 'en', 'language in English');
```

Comment:
As `language` is being removed, let's try to translate `user_id` instead. This will produce the same exception.